components/workbooks_crm/actions/find-or-create-order/find-or-create-order.mjs

```diff
@@ -0,0 +1,97 @@
+import app from "../../workbooks_crm.app.mjs";
+import utils from "../../common/utils.mjs";
+
+export default {
+  key: "workbooks_crm-find-or-create-order",
+  name: "Find or Create an Order",
+  description: "Returns and order if exists otherwise creates a new one.",
```

Comment:
HI @luancazarine there is no link because the docs are dynamic. Once you are loggedin you need to go to `Cofiguration -> Automation -> API Reference` in order to see the endpoint docs.